features/storage/TESTS/kvstore/direct_access_devicekey_test/main.cpp

```diff
@@ -50,7 +50,7 @@ static inline uint32_t align_down(uint64_t val, uint64_t size)
     return (((val) / size)) * size;
 }
 
-int  get_virtual_TDBStore_position(uint32_t conf_start_address, uint32_t conf_size, bool is_conf_tdb_internal,
+int  get_virtual_TDBStore_position(uint32_t conf_start_address, uint32_t conf_size, bool is_conf_tdbd_internal,
```

Comment:
The parameter doesn't seem to be used for anything actually.

Reply:
Agreed - I think this is a vestige from the old version of the test prior to https://github.com/ARMmbed/mbed-os/commit/722628be0250fbd09248d92e3e2720775ec0f84b . I'll push another commit to remove.

Reply:
Removed the unused parameter